code-security/admin_guide/book.yml

```diff
@@ -93,6 +93,8 @@ topics:
     file: add-terraform-enterprise-run-tasks.adoc
   - name: Connect VScode with Prisma Cloud Code Security
     file: connect-vscode.adoc
+- name: Supported Languages for Specific Domains on Prisma Cloud Code Security
```

Comment:
2 things:

1. Why not have a new section for IaC like we do for SCA? https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-code-security/scan-monitor/software-composition-analysis/supported-package-managers
2. Let's make this more searchable. How about `Supported domain specific languages (DSLs)`

Reply:
@tsmithv11 - the placement of this file looks fine here? Can rename the file.

cc: @Pubs-MV

Reply:
Yeah, it works for now. I'm saying in the future, we should consider an IaC page that includes the DSLs among a description.